docs/develop/01_sdk/02_cookbook/06_signCallback.md

```diff
@@ -0,0 +1,78 @@
+---
+id: signCallback
+title: SignCallback
+---
+
+import TsJsBlock from '@site/src/components/TsJsBlock';
+
+import SignCallback from '!!raw-loader!@site/code_examples/core_features/signCallback/signCallback.ts';
+import SignExtrinsicCallback from '!!raw-loader!@site/code_examples/core_features/signCallback/signExtrinsicCallback.ts';
+import GetStoreTxSignCallback from '!!raw-loader!@site/code_examples/core_features/signCallback/getStoreTxSignCallback.ts';
+
+Signing data involves using the private key and therefore needs to be secure.
+There are many different options how data could be sign.
+You might have the private key stored in memory and are therefore able to simply sign the data.
+This is the easiest option but also comes with higher security risk.
+Storing the private key on a separate device or inside a sandboxed application can increase security.
+But to enable these security options, we need a generic interface to talk to the signer.
+This is what the `SignCallback` does.
+
+The `SignCallback` defines an interface between the SDK and an arbitrary signing strategy.
+May it be a ledger, an air gaped phone or your browser extension.
+The interface is generic enough to support implementations for all these security measures.
+There are three different flavours of the `SignCallback`.
+
+## SignCallbacks Family
+
+There are three types of signing callbacks.
```

Comment:
Repetition of the two lines above.

Reply:
[9e73861](https://github.com/KILTprotocol/docs/pull/181/commits/9e738613d5307cdd06ee60bd310bf4505b7b36a3)